token-swap/program/src/processor.rs

```diff
@@ -605,6 +624,7 @@ impl PrintProgramError for SwapError {
             SwapError::ExceededSlippage => {
                 info!("Error: Swap instruction exceeds desired slippage limit")
             }
+            SwapError::InvalidWeight => info!("Provided weight outside of valid range (1-99)"),
```

Comment:
Isn't max weight 100?

Reply:
It is, thanks for the catch.  I've actually been wondering, why do we specify the same error message in two different places?  It would be easier to just take the error message defined in `error.rs` and log that instead to avoid copy-pasting.  I'll put that in so you can see, but feel free to reject it :-)

Reply:
No good reason that I know of -- I went through the token errors to see if there were any substantive differences and didn't find any.
So much simpler with your change 😍